### ISSUE TITLE
Visual Map not rendering line color (but it IS rendering correct color in label)[Bug] 

### ISSUE THREAD

**Issue by craigchandler** [apache/echarts#18629]
### Version

5.4.2

### Link to Minimal Reproduction

https://codepen.io/daemach/pen/MWegRvO

### Steps to Reproduce

See code pen.

### Current Behavior

Line color does not change, although the LABEL color is being mapped correctly according to visual map rules

### Expected Behavior

I expect the line color to change based on values in dimension 2 of series 0

### Environment

```markdown
- OS: Linux
- Browser: Firefox && Chrome
- Framework: javascript / angular
```

### Any additional comments?

see #13393

**Comment by craigchandler** [comment]
It should be noted that when the series type is 'bar' the bar color matches that defined by dimension not the plotted value

**Comment by helgasoft** [comment]
Agree, kind of weird. People want to do **line segments** coloring in ECharts, but get just **line points** (+label) coloring.
Solution is to use custom lines - see code example in #18634
